server/src/main/kotlin/org/itmodreamteam/myrest/server/service/table/TableService.kt

```diff
@@ -0,0 +1,20 @@
+package org.itmodreamteam.myrest.server.service.table
+
+import org.itmodreamteam.myrest.server.model.restaurant.RestaurantTable
+import org.itmodreamteam.myrest.shared.table.TableInfo
+import org.itmodreamteam.myrest.shared.table.TableView
+
+interface TableService {
+
+    fun addTable(restaurantId: Long, info: TableInfo): TableView
+
+    fun updateTable(tableId: Long, info: TableInfo): TableView
+
+    fun getTable(tableId: Long): TableView
+
+    fun removeTable(tableId: Long): TableView
+
+    fun getRestaurantTables(restaurantId: Long): List<TableView>
+
+    fun toTableView(table: RestaurantTable): TableView
```

Comment:
IMHO converter methods should not be in service level classes. It's usually different layer because of SRP.
Service does business logic as well as converting entities.
Why is it bad? Because 
1. Entity to be converted (say it "xxx") can contain references to other entities (say it "xxxRef") which should be converted as well. So in you XxxService you need to inject XxxRefService. This can lead to cycle dependencies between services which is pretty ok for spring, but may be a problem in tests (X requires Y, Y requires Z, Z requires X -> how to test X?). Cycle dependencies are evil but they are very common for converters so our services will have cycle dependencies even not for doing real business logic but for converting entities.
2. (this is not really relevant for us, but anyway). What if we would want to have different implementation of TableService. We will need to implement the same logic for conversion what will lead to code duplication.

Reply:
Created an issue #21